pkg/ecdsa/tss/protocol_ready.go

```diff
@@ -97,3 +143,22 @@ func readyProtocol(
 		return fmt.Errorf("unexpected context error: [%v]", ctx.Err())
 	}
 }
+
+func memberIDToAddress(
```

Comment:
I think we should handle it as a function for `MemberID` and not pass `pubKeyToAddressFn`.

Reply:
This would be a good idea if there was only one way to convert pub key to address. Then, `MemberID` is just a simple value object. Here, we can have multiple strategies.

Reply:
> Here, we can have multiple strategies

We don't have them yet, right?

Reply:
The entire architecture is organized in a way supporting those strategies. We have two: ethereum and local. We can't ignore `chain.Handle` abstraction and work with ethereum implementation directly. This way, why to have `chain.Handle` at all - see [this](https://github.com/keep-network/keep-ecdsa/pull/654#discussion_r549380907) answer.

Importing anything `go-ethereum` from anything else then `ethereum.go` is bad practice, that's all.